tools/install_base.sh

```diff
@@ -759,6 +785,9 @@ if ls $PA_ROOT/default/addons/*.jar > /dev/null 2>&1; then
         if [[ "$OLD_VERSION" != "$NEW_VERSION" ]]; then
            rm -f  $PA_ROOT/default/addons/*$OLD_VERSION*.jar
            rm -f  $PA_ROOT/default/addons/*/*$OLD_VERSION*.jar
+           OLD_VERSION_MAJOR=$(compute-major-minor OLD_VERSION)
+        else
+           OLD_VERSION_MAJOR=NEW_VERSION_MAJOR
```

Comment:
I don't understand these OLD_VERSION_MAJOR= 

OLD_VERSION_MAJOR is nowhere used

Reply:
true, I used this before I decided to rely on looking at the config/rm/settingsini configuration, I can remove it.